test/controllers/omniauth_callbacks_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
Remove `test` folder.
Please add Rspec and configure generators https://stackoverflow.com/a/22400845/3753977